eth/api_backend.go

```diff
@@ -279,7 +279,10 @@ func (b *EthAPIBackend) SubscribeLogsEvent(ch chan<- []*types.Log) event.Subscri
 }
 
 func (b *EthAPIBackend) SendTx(ctx context.Context, signedTx *types.Transaction) error {
-	return b.eth.txPool.Add([]*types.Transaction{signedTx}, true, false)[0]
+	if locals := b.eth.localTxTracker; locals != nil {
```

Comment:
Why copy it to `locals`?

Reply:
What do you mean -- it's the api backend, it's where we tag them as locals? Did I do something wrong?